<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSettingsRoom</class>
 <widget class="QDialog" name="DialogSettingsRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings Room</string>
  </property>
  <widget class="QTextEdit" name="textEditPublicKeysClients">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>481</width>
     <height>251</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelPubKeysClient">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>141</width>
     <height>15</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>15</height>
    </size>
   </property>
   <property name="text">
    <string>Public Keys Requests</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditKeyRoomAES">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelKeyRoom">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>549</width>
     <height>15</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>15</height>
    </size>
   </property>
   <property name="text">
    <string>Key Room ( AES-256 )</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGenAES">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>77</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Gen new AES-256 keys</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>370</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>DialogSettingsRoom</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
